.oscal/ssdf.catalog.oscal.yaml

```diff
@@ -0,0 +1,1058 @@
+            - name: statement
+              props:
+                - name: label
+                  value: RV.2.1
+              prose: >-
+                Analyze each vulnerability to gather sufficient information
+                about risk to plan its remediation or other risk response.
+            - name: statement
+              props:
+                - name: label
+                  value: RV.2.2
+              prose: Plan and implement risk responses for vulnerabilities.
+          props:
+            - name: label
+              value: RV.2
+          title: Assess, Prioritize, and Remediate Vulnerabilities
+        - id: rv3
+          parts:
+            - name: overview
+              prose: Help reduce the frequency of vulnerabilities in the future.
+            - name: statement
+              props:
+                - name: label
+                  value: RV.3.1
+              prose: >-
+                Analyze identified vulnerabilities to determine their root
+                causes.
+            - name: statement
+              props:
+                - name: label
+                  value: RV.3.2
+              prose: >-
+                Analyze the root causes over time to identify patterns, such as
+                a particular secure coding practice not being followed
+                consistently.
+            - name: statement
+              props:
+                - name: label
+                  value: RV.3.3
+              prose: >-
+                Review the software for similar vulnerabilities to eradicate a
+                class of vulnerabilities, and proactively fix them rather than
+                waiting for external reports.
+            - name: statement
+              props:
+                - name: label
+                  value: RV.3.4
+              prose: >-
+                Review the SDLC process, and update it if appropriate to prevent
+                (or reduce the likelihood of) the root cause recurring in
+                updates to the software or in new software that is created.
+          props:
+            - name: label
+              value: RV.3
+          title: Analyze Vulnerabilities to Identify Their Root Causes
+  metadata:
+    links:
+      - rel: canonical
+        href: >-
+          https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-218.pdf
+    props:
+      - name: keywords
+        value: >-
+          secure software development,Secure Software Development Framework
+          (SSDF),secure software development practices,software
+          acquisition,software development,software development life cycle
+          (SDLC),software security
+    roles:
+      - id: creator
+        title: Creator of the Original Content
+      - id: prepared-by
+        title: Author of Content in OSCAL Format
+    title: >-
+      Secure Software Development Framework (SSDF) Version 1.1: Recommendations
+      for Mitigating the Risk of Software Vulnerabilities
```

Comment:
As discussed before can we meet with the group and discuss the larger picture and next steps? I think SSDF is a good choice but we should sync on what that means for us next and how this work plays out.